CHANGELOG.asciidoc

```diff
@@ -35,6 +35,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 * Dockerized all test environment for .NET, JavaScript, Python, Go, and Python-based tests for Console, and added Docker as a build requirement.
 * Async operations in .NET can now be cancelled. This however does not cancel work that is already happening on the server.
 * Bumped to `snakeyaml` 1.32 to fix security vulnerability.
+* Added user agent to web socket handshake in dotnet driver. Can be controlled by EnableUserAgentOnConnect in `ConnectionPoolSettings`. It is enabled by default.
```

Comment:
Please either change this from _dotnet driver_ to _the Gremlin.Net driver_ or _the .NET driver_.
And _EnableUserAgentOnConnect_ should also be marked as code, so: \`EnableUserAgentOnConnect\`